common/neon/transaction_model.py

```diff
@@ -1,44 +1,70 @@
 from __future__ import annotations
 
```

Comment:
Please run `black`-utility on the source code

Reply:
done